13/06/2022 19:28:46 The log file has been set-up.
13/06/2022 19:28:46 Worker process running with:
13/06/2022 19:28:46  - Job Name: process
13/06/2022 19:28:46  - PID: 12246
13/06/2022 19:28:46  - Threads: 1
13/06/2022 19:28:46  - Node: rosetta6
13/06/2022 19:28:46  - System: Linux
13/06/2022 19:28:46  - Machine: x86_64
13/06/2022 19:28:46  - Timeout Limit: 356400
13/06/2022 19:28:46  - Process: [memmap(['/Mounts/rbg-storage1/users/ayed/shapepipe_data/inputs/flat_SNR/mid_density/L512/shapepipe_run_2022-06-13_17-23-33/merge_starcat_runner/output/full_starcat-0000000.fits'],
       dtype='<U167')]
13/06/2022 19:28:46  - Running module: mccd_plots_runner
13/06/2022 19:28:46 TOTAL e1 residual RMSE: 6.626422e-03

13/06/2022 19:28:46 TOTAL e2 residual RMSE: 5.664640e-03

13/06/2022 19:28:46 TOTAL R2 residual RMSE: 1.461046e-01

13/06/2022 19:28:52 Bins: e1 residual RMSE: 0.003527

13/06/2022 19:29:02 Bins: e2 residual RMSE: 0.003062

13/06/2022 19:29:12 Bins: R2 residual RMSE: 0.012719

13/06/2022 19:29:21 [('ra', 'dec', 'g1', 'g2', 'psf_g1', 'psf_g2', 'w')]
13/06/2022 19:29:21 [('ra', 'dec', 'g1', 'g2', 'psf_g1', 'psf_g2', 'w')]
13/06/2022 19:29:21 [('ra', 'dec', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
13/06/2022 19:29:21 [('ra', 'dec', 'g1', 'g2', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
13/06/2022 19:29:21 [('ra', 'dec', 'sigma', 'psf_g1', 'psf_g2', 'psf_sigma', 'w')]
13/06/2022 19:29:21  > Computing rho statistics...
13/06/2022 19:29:21 ERROR: Cannot use given fields: ["{'coords':", "'spherical',", "'metric':", "'Euclidean',", "'sep_units':", "'arcmin',", "'bin_type':", "'Log'}"]
Traceback (most recent call last):
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 180, in _run_worker
    with_timeout(self.worker_dict['timeout'], self.w_log.name)(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/timeout.py", line 42, in inner
    return decorated(*args, **kwargs)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 194, in _worker_execution
    self._run_module()
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/pipeline/worker_handler.py", line 215, in _run_module
    self._stdout, self._stderr = self._module_runner(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_plots_runner.py", line 140, in mccd_plots_runner
    mccd_plots.rho_stats(
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_package/mccd_plot_utilities.py", line 1508, in rho_stats
    rho_results = [
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/shapepipe/modules/mccd_package/mccd_plot_utilities.py", line 1509, in <listcomp>
    rho_stat(stilecat, config=TreeCorrConfig)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/sys_tests.py", line 819, in __call__
    return self.getCF('gg', new_data, new_data2, new_random, new_random2,
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/sys_tests.py", line 543, in getCF
    results = stile.ReadTreeCorrResultsFile(output_file)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/treecorr_utils.py", line 105, in ReadTreeCorrResultsFile
    return stile_utils.FormatArray(output, fields=fields)
  File "/Mounts/rbg-storage1/users/ayed/miniconda3/envs/shapepipe/lib/python3.9/site-packages/stile/stile_utils.py", line 90, in FormatArray
    raise RuntimeError('Cannot use given fields: '+str(fields))
RuntimeError: Cannot use given fields: ["{'coords':", "'spherical',", "'metric':", "'Euclidean',", "'sep_units':", "'arcmin',", "'bin_type':", "'Log'}"]
13/06/2022 19:29:21 The log file has been closed.
